spec/factories/invoices.rb

```diff
@@ -1,6 +1,6 @@
 FactoryBot.define do
   factory :invoice do
-    customer_id 
-    status { [1,2,3].sample }
+    customer_id { nil }
+    status { [0,1,2].sample }
```

Comment:
3 was not a valid status, needed to start enum from 0

Reply:
That makes sense since these are referencing enum index numbers